Source: Thomas Anderson <tanderson@caltech.edu>
Upstream: e-mailed
Reason: multiarch

--- a/configure.in
+++ b/configure.in
@@ -101,7 +101,7 @@ AC_LIBTOOL_WIN32_DLL
 AC_PROG_LIBTOOL
 AC_PROG_INSTALL
 AC_PROG_LN_S
-AC_PATH_PROG(AR, ar, no)
+AC_CHECK_TOOL(AR, ar, no)
 if [[ $AR = "no" ]] ; then
     AC_MSG_ERROR("Could not find ar - needed to create a library")
 fi
